src/main/java/seedu/duke/ui/Formatter.java

```diff
@@ -253,10 +258,10 @@ private static String encloseRow(String message) {
      * Returns the number of ascii code that appear within the message. Also match the start and end of index of each
      * color.
      *
-     * @param message String of the message to check.
+     * @param message                     String of the message to check.
      * @param coloredStringStartIndexList Arraylist of index to mark the start of a color.
-     * @param coloredStringEndIndexList Arraylist of index to mark the end of a color.
-     * @param stringColorList Arraylist of string to store the color present.
+     * @param coloredStringEndIndexList   Arraylist of index to mark the end of a color.
+     * @param stringColorList             Arraylist of string to store the color present.
```

Comment:
All of our Javadocs was @param parameter definition. There was no additional whitespace to align the columns. If we are doing this, should we then standardize for all JavaDocs? Would it be easier to just have a single space?

Reply:
I didnt realise this. I followed the format from addressbook to make it tidy and easy on the eyes. Jx and I used whitespaces when implementing the classes in the first place. I think its a minor matter as long its readable.